test/integration/targets/pids/aliases

```diff
@@ -0,0 +1 @@
+unsupported
```

Comment:
Why can't this test be run in CI? It would be especially useful to see whether it works everywhere.

Reply:
The easiest is to try it out :)

Simply put `shippable/posix/group3` in here instead of `unsupported`, and see what happens. Maybe you will get some test failures, but at least then you get an idea why they fail where (for example, `pidof` might not be available somewhere)

Maybe you should also add some more tests which probably match processes running on the machine (like check for `python`, `python2`, `python3`) to see whether something bad will happen if something is found.